rosbag2_performance/rosbag2_performance_benchmarking_msgs/package.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>rosbag2_performance_benchmarking_msgs</name>
+  <version>1.0.0</version>
+  <description>A package containing rosbag2 performance benchmarking specific messages.</description>
+  <maintainer email="opensource@apex.ai">Apex AI, Inc.</maintainer>
```

Comment:
Please add "openrobotics" group for maintainers as in another packages in rosbag2

Reply:
done